lcc_lokavaluto_app_connection/services/partner_services.py

```diff
@@ -65,7 +65,41 @@ def credit_requests(self, partner_credit_requests_get_param):
     )
     def pending_topup(self):
         backend_keys = request.params["backend_keys"]
-        return self.env["account.invoice"]._get_pending_credit_requests(backend_keys)
+        return self.env["account.invoice"]._get_pending_credit_requests(
```

Comment:
Here the pending credit requests are not sent in chronological order. Are we sure that LokAPI or Monujo will do the job so that the requests are displayed chronologically in the app?

Reply:
From what I understand, the models from which the sale order and account invoice are inherited already order the elements by date. However, we added a more explicit filter to make the code more readable.